arrays/arrays.c

```diff
@@ -31,6 +31,8 @@ Array *create_array(int capacity)
   // Allocate memory for elements
   // we need to malloc times the size of the thing we're pointing to (bc elements is being declared via double pointer).
   arr->elements = malloc(sizeof(char *) * capacity);
```

Comment:
```suggestion
  arr->elements = malloc(sizeof(char *) * capacity);
```
don't use malloc - it does not pass the tests